client/data/media-storage/with-media-storage.js

```diff
@@ -3,6 +3,17 @@ import { useSelector } from 'react-redux';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import useMediaStorageQuery from './use-media-storage-query';
 
+export {
+	KB_IN_BYTES,
+	MB_IN_BYTES,
+	GB_IN_BYTES,
+	TB_IN_BYTES,
+	PB_IN_BYTES,
+	EB_IN_BYTES,
+	ZB_IN_BYTES,
+	YB_IN_BYTES,
+} from './use-media-storage-query';
```

Comment:
Do we need all those constants?  Could we check if `storage_used_bytes` is greater than half of `max_storage_bytes` instead?

Reply:
I updated the pr. I have also updated the description as well! Thanks!